darwin/exporter/formats/nifti.py

```diff
@@ -160,6 +160,8 @@ def build_output_volumes(
         )
         # Builds output volumes per class
         volume_dims, pixdims, affine, original_affine = process_metadata(slot.metadata)
+        if not class_names_to_export:
+            class_names_to_export = ["empty"]
```

Comment:
This basically says: If there are no polygons for export, create an empty NifTI volume as though there was a polygon with class name `empty`

I'm not very happy with just referring to this as `empty`, so open for suggestions!

This also means that if a user exports NifTI files with mask annotations but no polygons, there will be two files output. This doesn't seem ideal. Should we first check if there are raster annotations and skip this if they are present?